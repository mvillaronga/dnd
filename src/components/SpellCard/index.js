import React from 'react'

const SpellCard = ({name, level, onClick}) => {

    return <div onClick={onClick} >
        {name} - {level}
    </div>
}

export default SpellCard