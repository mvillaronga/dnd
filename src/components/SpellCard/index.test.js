import React from 'react';
import SpellCard from '../SpellCard/index';
import renderer from 'react-test-renderer';

test('Default Spell Card Rendering', () => {
    const component = renderer.create(
      <SpellCard name="Some Sample Spell" level="1" />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });