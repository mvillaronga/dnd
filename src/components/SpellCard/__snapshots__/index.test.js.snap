// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Spell Card Rendering 1`] = `
<div>
  Some Sample Spell
   - 
  1
</div>
`;
